api/backup.go

```diff
@@ -148,7 +148,7 @@ func (api *api) RestoreBackup(unlockPassword string, r io.Reader) error {
 		return fmt.Errorf("failed to create decrypted reader: %w", err)
 	}
 
-	tmpF, err := os.CreateTemp("", "nwc-*.bkp")
+	tmpF, err := os.CreateTemp(api.cfg.GetEnv().Workdir, "nwc-*.bkp")
```

Comment:
should the file be renamed to `albyhub-` ?

and maybe we can make a sub directory?

Reply:
Yes I wanted to change this to albyhub- as well, also the backup file. But would it have any side effects? cc @rolznz

Reply:
aren't you creating a brand new folder? why are there side effects?

Reply:
Oh. This temp file is only used in this backup process. So if you rename it here, just check it is not used elsewhere

Reply:
@im-adithya you didn't rename the file?